### DISCUSSION TITLE
when high precision mouse options are enabled,osu still can't bypass the mouse acceleration

### DISCUSSION THREAD

**Discussion by ZhuoBaiOof** [ppy/osu#27732] [Q&A]
### Type

Other

### Bug description

在我的huawei matepad11 上，当我玩osu!lazer时，即使我打开了高精度鼠标选项，鼠标加速仍然会工作
它的系统版本是harmony os 4(基于aosp 12)

### Version

2024.312.1-lazer

### Logs

[logs.zip](https://github.com/ppy/osu/files/14757423/logs.zip)

**Comment by Walavouchey** [comment]
thank you for the report. these types of issues are likely hardware-specific and therefore hard to investigate on our end

using a mouse on an android tablet is also a very niche use case, so it is not currently a high priority to support

either you or someone else with similar hardware who can reproduce this issue might be able to debug and provide more information regarding the root cause

**Reply by ZhuoBaiOof** [reply]
所以在其他的设备上不存在此问题吗？

**Reply by ZhuoBaiOof** [reply]
好吧，我在我的note 12 turbo （android 14）上测试了一下，并没有复现这个问题

**Reply by ZhuoBaiOof** [reply]
你们可以测试一下是android 12的问题吗？

**Reply by Walavouchey** [reply]
so the issue has been found in:

- huawei matepad11 (harmony os 4, based on android 12)
- moto g play (2021) (android 11)

if you're able to provide english translations along with what you type in chinese (even if they're machine translations), that would be convenient and help future people investigating this to follow along

**Comment by Joehuu** [comment]
Seems to work on my Pixel 6a (Android 14) using the included USB C to A adapter and a random mouse. With high precision mouse disabled, I see the Android cursor and the osu! cursor, but with it off, the Android cursor disappears. Does the same happen to you?

Edit: Can reproduce on moto g play (2021) (Android 11), will see if bottom changes work.

~Might be resolved once the changes in https://github.com/ppy/osu-framework/pull/6105 are accepted, but is not currently a priority.~ Doesn't fix.

**Reply by ZhuoBaiOof** [reply]
yes

**Reply by ZhuoBaiOof** [reply]
那么在你的设备上，鼠标加速可以被绕过吗？
